package/upgrade/upgrade_manifests.sh

```diff
@@ -489,24 +489,39 @@ wait_longhorn_instance_manager_aio() {
   # Wait for instance-manager (aio) pods upgraded to new version first.
   kubectl get nodes.longhorn.io -n longhorn-system -o json | jq -r '.items[].metadata.name' | while read -r node; do
     echo "Checking instance-manager (aio) pod on node $node..."
-    while [ true ]; do
-      im_count=$(kubectl get instancemanager.longhorn.io --selector=longhorn.io/node=$node,longhorn.io/instance-manager-type=aio,longhorn.io/instance-manager-image=$im_image_checksum -n longhorn-system -o json | jq -r '.items | length')
-      if [ "$im_count" != "1" ]; then
-        echo "instance-manager (aio) (image=$im_image) count is not 1 on node $node, will retry..."
-        sleep 5
-        continue
-      fi
+    check_instance_manager $node $im_image $im_image_checksum "v1"
 
-      im_status=$(kubectl get instancemanager.longhorn.io --selector=longhorn.io/node=$node,longhorn.io/instance-manager-type=aio,longhorn.io/instance-manager-image=$im_image_checksum -n longhorn-system -o json | jq -r '.items[0].status.currentState')
-      if [ "$im_status" != "running" ]; then
-        echo "instance-manager (aio) (image=$im_image) state is not running on node $node, will retry..."
-        sleep 5
-        continue
-      fi
+    v2EngineEnabled=$(kubectl get settings.harvesterhci.io longhorn-v2-data-engine-enabled -o yaml | yq -e e '.value' -)
```

Comment:
@Vicente-Cheng I'm testing v1.4.2 to v1.5-head. This line throws error like following if there is no value field:

```
null
Error: no matches found
```

I can fix this by removing `-e`. Before I filing a PR, I would like to know is it ok to remove it? Thanks.